ext/nio4r/monitor.c

```diff
@@ -117,6 +119,34 @@ static VALUE NIO_Monitor_initialize(VALUE self, VALUE io, VALUE interests, VALUE
     return Qnil;
 }
 
+static VALUE NIO_Monitor_setInterests(VALUE self, VALUE interests){
+    //Check Whether the Monitor is already Closed?
+    if(NIO_Monitor_is_closed(self) == Qfalse){
```

Comment:
I use that because the only situation where we execute that block is there
monitor is not closed. So that is why use Qfalse. thought there might be
some other values returning from the isClosed other than QTrue and Qfalse.
(like Qnil)
.

Upekshe Jayasekera
Undergraduate
Department of Computer Science and Engineering
University of Moratuwa
Sri Lanka

On 2 June 2015 at 12:36, Tony Arcieri <notifications@github.com> wrote:

> In ext/nio4r/monitor.c
> <https://github.com/celluloid/nio4r/pull/73#discussion_r31497104>:
>
> > @@ -117,6 +119,34 @@ static VALUE NIO_Monitor_initialize(VALUE self, VALUE io, VALUE interests, VALUE
> >      return Qnil;
> >  }
> >
> > +static VALUE NIO_Monitor_setInterests(VALUE self, VALUE interests){
> > +    //Check Whether the Monitor is already Closed?
> > +    if(NIO_Monitor_is_closed(self) == Qfalse){
>
> Yep
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/celluloid/nio4r/pull/73/files#r31497104>.
>